src/components/CalendarComponents/categoriesBar.vue

```diff
@@ -71,7 +71,6 @@ export default class CategoriesBar extends Vue {
         display: flex;
         justify-content: space-between;
         align-items: center;
-        width: 90%;
```

Comment:
Why take it off?

Reply:
Because it causes the categories bar to be smaller than the calendar